### DISCUSSION TITLE
How to Pass Parameters to Airflow DAG from UI in the Latest Version 2.8.2

### DISCUSSION THREAD

**Discussion by Venkatesh748** [apache/airflow#39221] [Q&A]
Earlier we are using Airflow V2.5.0. In this version we can pass parameters to Airflow DAG in the form of key values by choosing the Trigger DAG w/config option in trigger button in the Airflow UI.
![image](https://github.com/apache/airflow/assets/53960890/a821b638-b838-459a-aa34-0a2e732dcbcc)

Now we upgraded the Airflow Version to 2.8.2.In the upgraded version I couldn't see that option(w/config) while triggering the DAG. It seems like it is removed in the latest version. I know we can pass the parameters using CLI but my use case is I need to pass parameters through Airflow UI only. 
Could some please let me know know how to pass parameters in the latest Airflow versions from Airflow UI.

Thank You.

**Comment by phi-friday** [comment]
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#show-trigger-form-if-no-params

set `AIRFLOW__WEBSERVER__SHOW_TRIGGER_FORM_IF_NO_PARAMS=true`
> New in version 2.7.0.
